web/leaflet/MarkerCluster.Default.css

```diff
@@ -1,60 +1,60 @@
 .marker-cluster-small {
-	background-color: rgba(181, 226, 140, 0.6);
```

Comment:
All the files in leaflet, are from the leaflet disitribution so I think it is ok to leave them as is, if we upgrade the version of leaflet we will just copy the new version over the current files and not apply any pretifier to it.

Reply:
This problem can be solved easily.